utils/src/main/kotlin/LicenseFilenamePatterns.kt

```diff
@@ -61,5 +61,16 @@ data class LicenseFilenamePatterns(
                 "readme*"
             )
         )
+
+        private var instance: LicenseFilenamePatterns = DEFAULT
```

Comment:
Unrelated to this commit, but I just noticed that the docs for `licenseFilenames` are missing the sentence "The patterns are supposed to be used case-insensitively". As you are working on this, can you add this in one of your PRs?